helm/TESTS.md

```diff
@@ -9,5 +9,5 @@ helm plugin install https://github.com/helm-unittest/helm-unittest.git
 Inside `apim3` directory, run :
 
 ```shell
-helm unittest -3 -f 'tests/**/*.yaml' .
+helm unittest -f 'tests/**/*.yaml' .
```

Comment:
with -3 it's not working on my side 🤷‍♂️

Reply:
it was used with older version of helm-unit plugin